conf/script/ha_setup

```diff
@@ -122,6 +124,15 @@ replace_node() {
 
 # Helping function
 
+add_permission() {
+    id hacluster && {
```

Comment:
A question - At no point of time, root user is supposed to write a log to this file?
If not then we are fine. Has testing been done thoroughly?

Reply:
Tested with RPM

changed some approch here
- metadata is static method and call independently and not access file
- added log permission to hacluster user
so log have both hacluster and root permission


```
[root@ssc-vm-0491 708945]# getfacl /var/log/seagate/cortx/ha/
getfacl: Removing leading '/' from absolute path names
# file: var/log/seagate/cortx/ha/
# owner: root
# group: root
user::rwx
user:hacluster:rwx
group::r-x
mask::rwx
other::r-x
```